### PR TITLE
avoid some possible risk

### PR DESCRIPTION
**Pull request by KAMIJYOUDOUMA**
### Ⅰ. Describe what this PR did
1.prevent users from changing constant value 
2.add some NPE check

### CODE REVIEW

File: common/src/main/java/io/seata/common/exception/FrameworkErrorCode.java [hunk 1]
```diff
@@ -94,28 +94,39 @@
     /**
      * 未定义错误
      */
-    UnknownAppError("10000", "unknown error", "内部错误"),
-    ;
+    UnknownAppError("10000", "unknown error", "内部错误"),;
```
Comment (CoffeeLatte007) [inline common/src/main/java/io/seata/common/exception/FrameworkErrorCode.java]: ,
;

File: core/src/main/java/io/seata/core/rpc/netty/RmMessageListener.java [hunk 2]
```diff
@@ -94,7 +94,10 @@ private void handleBranchCommit(long msgId, String serverAddress,
             resultMessage = (BranchCommitResponse)handler.onRequest(branchCommitRequest, null);
             sender.sendResponse(msgId, serverAddress, resultMessage);
         } catch (Exception e) {
-            LOGGER.error(FrameworkErrorCode.NetOnMessage.errCode, e.getMessage(), e);
+            LOGGER.error(FrameworkErrorCode.NetOnMessage.getErrCode(), e.getMessage(), e);
+            if(resultMessage == null){
```
Comment (CoffeeLatte007) [inline core/src/main/java/io/seata/core/rpc/netty/RmMessageListener.java]: maybe resultMessage already is not null

File: tcc/src/main/java/io/seata/rm/tcc/api/BusinessActionContext.java [hunk 3]
```diff
@@ -68,7 +68,7 @@ public Object getActionContext(String key) {
      * @return the branch id
      */
     public long getBranchId() {
-        return branchId!=null?Long.valueOf(branchId):-1;
+        return branchId!=null?Long.parseLong(branchId):-1;
```
Comment (CoffeeLatte007) [inline tcc/src/main/java/io/seata/rm/tcc/api/BusinessActionContext.java]: code format . branchId != null ? Long.parseLong(branchId) : -1;

File: core/src/main/java/io/seata/core/rpc/netty/RmMessageListener.java (hunk 2 above)
Comment (KAMIJYOUDOUMA) [reply core/src/main/java/io/seata/core/rpc/netty/RmMessageListener.java]: > maybe resultMessage already is not null

`handler.onRequest` may throw a `RuntimeException`,then `resultMessage` will be null.

Comment (CoffeeLatte007) [reply core/src/main/java/io/seata/core/rpc/netty/RmMessageListener.java]: > > maybe resultMessage already is not null
> 
> `handler.onRequest` may throw a `RuntimeException`,then `resultMessage` will be null.

yes

**Pull request merged**